components/woo/shipping-method.php

```diff
@@ -480,6 +480,71 @@ public function init_settings_fields()
 				'description' => __( 'Enter standard phone number for sender.', 'shipcloud-for-woocommerce' ),
 				'desc_tip'    => true,
 			),
+			'banking_information' => array(
+				'title'       => _x(
+					'Banking information',
+					'Backend: Title of the settings section',
+					'shipcloud-for-woocommerce'
+				),
+				'type'        => 'title',
+				'description' => sprintf( __( 'Fill in defaults for cash on deliveryŝ.', 'shipcloud-for-woocommerce' ) ),
```

Comment:
`deliveryŝ` <-- that's a typo right? The last character should probably not be there :grin: